### ISSUE TITLE
Focus outline is missing

### ISSUE THREAD

**Issue by radoslavzeman** [bratislava/marianum.sk#328]
Please add outline ("focus ring") when interactive elements are keyboard focused.

Outline is important for accessibility. Now we do not display it at all.
